source/nodejs/adaptivecards-designer/src/containers/host-container.ts

```diff
@@ -3,17 +3,28 @@
 import { GlobalSettings, GlobalRegistry, CardObjectRegistry, CardElement, Action, HostConfig, SerializationContext, Version, Versions } from "adaptivecards";
 import * as hostConfig from "../hostConfigs/sample.json";
 
+export enum ContainerSize {
+    Small = "Small",
```

Comment:
Please make sure you are using lowercases for large/medium/small in $host

![image](https://user-images.githubusercontent.com/6290692/186489917-ec658f9b-197d-45f0-8fb1-ed9da66f7385.png)